src/react/ui-elements/ErrorHandlerModal.tsx

```diff
@@ -25,15 +25,27 @@ const ErrorHandlerModal = () => {
   }
 
   return (
-    <Modal
-      close={close}
-      footer={<Button variant="primary" onClick={close} label="Close" />}
-      isOpen={true}
-      title="Error"
-    >
-      {errorMessage}
-    </Modal>
+    <DumbErrorModal errorMessage={errorMessage} isOpen={true} close={close} />
```

Comment:
If we set `isOpen={true}` does it mean the Modal will be already open?

Reply:
It was like that before, I just extracted the JSX into a separate component in order to reuse it without the redux-store logic :) And to answer your question - no, it's not going to be always open, because of this code couple of lines before (again, I haven't written that - it was already there) :


> const showError = useSelector(
>    (state: AppState) =>
>      !!state.uiErrors.errorMsg && state.uiErrors.errorType === 'byModal',
>  );
...
if (!showError) {
    return null;
}